src/api/itemGeolocation.ts

```diff
@@ -0,0 +1,59 @@
+import { Item, ItemGeolocation, UUID } from '@graasp/sdk';
+
+import { PartialQueryConfigForApi } from '../types';
+import { verifyAuthentication } from './axios';
+import {
+  buildDeleteItemGeolocationRoute,
+  buildGetItemGeolocationRoute,
+  buildGetItemsInMapRoute,
+  buildPutItemGeolocationRoute,
+} from './routes';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getItemGeolocation = async (
+  { API_HOST, axios }: PartialQueryConfigForApi,
+  id: UUID,
+) =>
+  axios
+    .get<ItemGeolocation>(`${API_HOST}/${buildGetItemGeolocationRoute(id)}`)
+    .then(({ data }) => data);
+
+export const putItemGeolocation = async (
+  payload: { itemId: Item['id'] } & Pick<ItemGeolocation, 'lat' | 'lng'>,
```

Comment:
If you put the geolocation in a special key it will make it easier to pass it as the body object without having to repeat the key-values.